### ISSUE TITLE
[Q] / [Feature] ShowCMD suggestions  / history displayed list

### ISSUE THREAD

**Issue by maoamid** [pekdon/pekwm#76]
Hi,
Is it possible to customize ShowCmd window to display in addition a list with potential commands / or even generated from history.
It's something like autocomplete enabled but also with a list of potential commands. It can be very useful when you don't know exactly a name of a command, or when you run repetitive tasks.

**Comment by pekdon** [comment]
I would assume it should be have something like **ShowSearchDialog** does?

It is not currently possible but sounds like a reasonable feature to have for sure!

**Comment by maoamid** [comment]
Yes, something like that. Right now I use rofi for that, but if PekWM can do this, it will be cool. Not something very customizable, just a simple list, with a simple option defining the max number of suggestions to be displayed, I think can be a good start.
Have a great weekend @pekdon
